app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -88,22 +95,26 @@ def add_claimant_info(user)
         'dob' => user.birth_date
       }
     ).except!('vaFileNumber')
+  end
 
-    update(form: updated_form.to_json)
+  def add_office_location(updated_form)
+    @office_location = check_office_location
+    updated_form['veteranInformation']&.merge!({ 'regional_office' => @office_location })
   end
 
   def send_to_vre(user)
     prepare_form_data
-    office_location = check_office_location
-
-    email_addr = REGIONAL_OFFICE_EMAILS[office_location] || 'VRE.VBACO@va.gov'
-    VeteranReadinessEmploymentMailer.build(user, email_addr).deliver_now if user.present?
 
     upload_to_vbms
 
+    @office_location = check_office_location if @office_location.nil?
+
+    email_addr = REGIONAL_OFFICE_EMAILS[@office_location] || 'VRE.VBACO@va.gov'
+    VeteranReadinessEmploymentMailer.build(user, email_addr).deliver_now if user.present?
+
```

Comment:
@LindseySaari I only changed one thing since your review/approval.  I changed the order of operations so that we call `upload_to_vbms` before we send the email notification (so lines 97-100 moved down to line 110-114).  Please let me know if you have any questions.

Reply:
Ok, I think that makes sense